tests/ballerina-examples-test/src/main/resources/run.sh

```diff
@@ -48,10 +48,6 @@ exclude=("proto-to-ballerina"
         "streaming-big-dataset"
         "docker-deployment"
         "kubernetes-deployment"
-        "error-handling"
-        "error-lifting"
-        "http-failover"
-        "xml-attributes"
```

Comment:
This test is failing with the following error.

```
Running tests
    ballerina-examples/xml-attributes:0.0.1
	[fail] testFunc:
	    error: Assertion Failed!: expected '<ns0:book xmlns:ns0="http://ballerina.com/aa" ns0:status="available" count="5"></ns0:book>' but found '<ns0:book xmlns:ns0="http://ballerina.com/aa" ns0:status="Not Available" count="5"></ns0:book>' {}
	    	at ballerina/test:createBallerinaError(assert.bal:42)
	    	ballerina/test:assertEquals(assert.bal:83)
	    	ballerina-examples/xml-attributes:0.0.1:testFunc(tests/xml_attributes_test.bal:28)

	0 passing
	1 failing
	0 skipped
```

Reply:
@hasithaa This is passing as following PR is merged
https://github.com/ballerina-platform/ballerina-lang/pull/13337